tkg/managementcomponents/types.go

```diff
@@ -79,7 +79,6 @@ type CoreManagementPluginsPackage struct {
 }
 
 type CoreManagementPluginsPackageValue struct {
```

Comment:
Do we still need this struct? Since it will be empty after DeployCLIPluginCRD is removed